src/main/java/de/teamlapen/vampirism/client/gui/SelectActionScreen.java

```diff
@@ -51,7 +54,7 @@ public boolean isEnabled() {
     };
 
     public SelectActionScreen() {
-        super(ICON_TEXTURE_WIDTH, ICON_TEXTURE_HEIGHT, 2298478591L, new TranslationTextComponent("selectAction"));
+        super(FactionPlayerHandler.get(Minecraft.getInstance().player).getCurrentFaction().getColor(), new TranslationTextComponent("selectAction"));
```

Comment:
Even though it is unlikely that a null pointer occurs here, it is usually not very clean and easy to debug, if we have such long statements in one line.
One alternative would e.g. be to make the color an constructor argument and retrieve the required color elsewhere